tests/py_test.py

```diff
@@ -545,10 +584,13 @@ def asserter(stmt):
         asserter(size == len(buf[offset:])-4)
         buf, offset = util.RemoveSizePrefix(buf, offset)
     if file_identifier:
+        # test again after removal of size_prefix
+        asserter(util.GetBufferIdentifier(buf, offset) == file_identifier)
+        asserter(util.BufferHasIdentifier(buf, offset, file_identifier=file_identifier))
         asserter(MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset))
-    else:
-        asserter(not MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset))
     monster = MyGame.Example.Monster.Monster.GetRootAsMonster(buf, offset)
+    # wrap moster objects _tab with one that verifies alignment
+    monster._tab = TestProxyTable(monster._tab, initialOffset)
```

Comment:
I'd prefer that we not modify the "gold" table in this way. Can you separate out the logic to make this a separate test path?

Reply:
It doesn't modify the table, but just wraps checks around it. But I see how if there are errors in the checking code, it could cause problems. I'll fix this. Does this mean you approve of the general test concept?